ArduPlane/quadplane.cpp

```diff
@@ -443,6 +443,15 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("ACRO_YAW_RATE", 13, QuadPlane, acro_yaw_rate, 90),
+
+    // @Param: LOIT_REV_SPD
+    // @DisplayName: QLOITER mode max reverse speed
+    // @Description: maximum backward airspeed in m/sec
+    // @Units: m/s
+    // @Range: 0 5
+    // @Increment: .1
+    // @User: Standard
+    AP_GROUPINFO("LOIT_REV_SPD", 14, QuadPlane, loit_rev_spd, 2),
```

Comment:
Do we want a value of 0 as "have no effect" so we don't change behavior on people who have tuned their aircraft?

Reply:
agreed.

Reply:
@WickedShell Thanks for catching all these issues;
 
I'd argue that the default should be 1 or 2 m/sec if a typical setup isn't stable when going backwards, since this feature would prevent attitude upsets.

OTOH, maybe I just don't know how to tune properly...
Regarding tuning of quadplanes, what is the typical procedure for tuning backwards flight in loiter mode, or if there isn't one what issues would this raise w.r.t. tuning?

All I have found so far about backward flight is a recommendation not to do it:, e.g.: https://discuss.ardupilot.org/t/tiltrotor-support-for-plane/8805/295?u=kd0aij